### ISSUE TITLE
Shebang should be /usr/bin/env python

### ISSUE THREAD

**Issue by jalla2000** [davorf/BlackBeanControl#36]
https://github.com/davorf/BlackBeanControl/blob/bf3a7543dac4f321ae72648b802985b4301e47ae/BlackBeanControl.py#L1

**Comment by deftdawg** [comment]
you mean python2 right?  I don't think it works with python3 which is likely were many systems would point "python" to

**Comment by jalla2000** [comment]
Yes, python2 would probably be correct if the code doesn't run on python3...

`/usr/bin/env python2`
